postman/Login_test

```diff
@@ -0,0 +1,75 @@
+{
+	"info": {
+		"_postman_id": "cdaee4f6-9167-4df8-8e54-3f10e5dcdab5",
+		"name": "Login_Form",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Post",
+			"item": [
+				{
+					"name": "API test",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Valid request\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"pm.test(\"email\", function () {\r",
+									"    pm.expect(pm.response.text()).to.include(\"lm.mishra56@gmail.com\");\r",
+									"});\r",
+									"\r",
+									"const jsonData = pm.response.json();\r",
+									"pm.test(\"Test data type of the response\", () => {\r",
+									"  pm.expect(jsonData.email).to.be.a(\"string\");\r",
+									"  pm.expect(jsonData.password).to.be.a(\"string\");\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoidW5kZWZpbmVkIHVuZGVmaW5lZCIsImVtYWlsIjoibG0ubWlzaHJhNTZAZ21haWwuY29tIiwiaWF0IjoxNjA5Njg4NzU0LCJleHAiOjE2MDk3MTAzNTR9.pFecTSTROnDj34Zxri14AfQ0X81_37Vmyh0OrAm5poI",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"email\":\"lm.mishra56@gmail.com\",\r\n\"password\":\"1234\"}\r\n",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://localhost:3500/auth/login",
+							"protocol": "http",
+							"host": [
+								"localhost"
+							],
+							"port": "3500",
+							"path": [
+								"auth",
+								"login"
+							]
+						}
+					},
+					"response": []
+				}
+			]
```

Comment:
You should include more than one request, demonstrating several use cases. Some of them could be

- 400 for invalid request
- Invalid email or password
And supporting test cases for them